### DISCUSSION TITLE
Version 1.1.26.1691 of copilot doesn't work

### DISCUSSION THREAD

**Discussion by shanjia-sdu** [community/community#19115] [Copilot Conversations] [answered]
When I do some editing, 
the following texts will appear in the "Event Log" window

Your Copilot experience is not fully configured, complete your setup.
		Copilot Settings   Dismiss

When I click the Copilot Settings link, nothing happens, and the copilot still doesn't work.

**Comment by rauzerfabiano** [comment, marked as answer]
1. Go to https://github.com/github-copilot/tp_signup
2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
3. In Suggestions matching public code; select Allow or Block.
4. Press Save and get started
5. Close or Reload the IDE or Code Editor
